Xwt.XamMac/Xwt.Mac/LinkLabelBackend.cs

```diff
@@ -102,7 +102,7 @@ public override void DisableEvent (object eventId)
 		void HandleClicked (object sender, EventArgs e)
 		{
 			ApplicationContext.InvokeUserCode (() => {
-				EventSink.OnNavigateToUrl (uri);
+				NSWorkspace.SharedWorkspace.OpenUrl (uri);
```

Comment:
Not sure this is correct. This means the OnNavigateToUrl event is no longer used. The LinkLabel can subscribe to this and take custom action.

Does this not get called?

https://github.com/mono/xwt/blob/d89687fd4d3885b42f3aeca2e82e0c54dc4fb7d9/Xwt/Xwt/LinkLabel.cs#L123

Reply:
Yeah, this breaks all the custom link implementation.

Reply:
Thx. I fixed it to override the desktop backend OpenUrl instead. Please review the latest.